src/main/kotlin/net/lumalyte/lg/infrastructure/services/ConfigServiceBukkit.kt

```
@@ -141,6 +158,83 @@ class ConfigServiceBukkit(private val config: FileConfiguration): ConfigService
+            physicalCurrencyItemValue = config.getInt("vault.physical_currency_item_value", 1),
+            physicalCurrencyRequireVaultChest = config.getBoolean("vault.physical_currency_require_vault_chest", true),
+            physicalDepositFee = config.getInt("vault.physical_deposit_fee", 0),
+            physicalWithdrawalFee = config.getInt("vault.physical_withdrawal_fee", 1),
+            physicalTransactionMinimum = config.getInt("vault.physical_transaction_minimum", 1),
+            physicalDailyWarCost = config.getInt("vault.physical_daily_war_cost", 10),
+            physicalWarDeclarationCost = config.getInt("vault.physical_war_declaration_cost", 100),
+            compressableBlocks = config.getStringList("vault.compressable_blocks")
+        )
+    }
+
+    private fun loadBedrockConfig(): BedrockConfig {
```

detekt (1.23.8)

[warning] The function loadBedrockConfig appears to be too complex based on Cyclomatic Complexity (complexity: 28). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CyclomaticComplexMethod)